observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,67 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+With this release, you can experience improved the performances of the eBPF agent, both in terms of CPU and memory. Additionally, the *NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two causes: when the Network Observability eBPF agent hashmap table is full, which cases the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]. 
+
+[id="network-observability-no-show-duplicates-1.6"]
+==== Duplicates no longer shown
+Previously, duplicated flows were stored as individual flows, and flagged as duplicate if there were multiple of the same kind of flows. You could choose to show the duplicated flows using 
+the *Duplicate flows* Query Option. Now instead of having multiple, individual flows, there is one flow with a list of matching *Interfaces* and *Interface directions*. Since the duplicated flows are merged in a list to a single flow data, Loki storage demand decreases significantly. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-quickfilternw-observe-network-traffic[Quick filters].
+
+[NOTE]
+===
+With this update, the *Interface* and *Interface Direction* fields in the *Network Traffic* table are renamed to *Interfaces* and *Interface Directions*, so any saved queries usi ng these fields need to be updated. 
+===
+
+[id="network-observability-ebpf-collection-filtering-1.6"]
+==== eBPF collection rule-based filtering
+You can use rule-based filtering to reduce the volume of created flows. When this option is enabled, the *Netobserv / Health* dashboard for eBPF agent statistics has the *Filtered flows rate* view. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
+
```

Comment:
I think something else is worth mentioning: https://issues.redhat.com/browse/NETOBSERV-1426 (I just added a release note text in it, if you want to check). Despite it's part of the Metrics API feature (more as a side task), IMO it should be mentioned apart because it offers new possibilities not necessarily correlated with the metrics API.

Reply:
Okay I added this release note text to the **eBPF collection rule-based filtering** release note.

Reply:
Now this:
> With this feature, you can add custom labels for any subnet that you would want to identify in your flows and metrics. It can also be used to better identify traffic from or to IPs that are external to the cluster. You can use rule-based filtering to reduce the volume of created flows. When this option is enabled, the *Netobserv / Health* dashboard for eBPF agent statistics has the *Filtered flows rate* view.

Reply:
Sorry the more I reflect on this comment the more I'm fairly certain you didn't mean for me to add it to the eBPF rule-based filtering note...since its part of the Custom Metrics API. I mention it in that note line 29.

Reply:
Yes I meant to mention it as an item of its own, but I think it's also good as you put it.